schemes/musica/musica_ccpp.F90

```diff
@@ -61,7 +73,8 @@ subroutine musica_ccpp_run(time_step, temperature, pressure, dry_air_density, co
                              surface_temperature, surface_albedo,                                    &
                              number_of_photolysis_wavelength_grid_sections,                          &
                              photolysis_wavelength_grid_interfaces, extraterrestrial_flux, &
-                             standard_gravitational_acceleration, errmsg, errcode)
+                             standard_gravitational_acceleration, cloud_area_fraction,             &
+                             air_pressure_thickness, errmsg, errcode)
```

Comment:
Not a big deal, but maybe we could match the spacing around the `&`?

Reply:
fixed!